Matchismo/Model/PlayingCardDeck.m

```diff
@@ -0,0 +1,35 @@
+//
+//  PlayingCardDeck.m
+//  Matchismo
+//
+//  Created by Yeliena Khaletska on 28.05.2024.
+//
+
+#import "PlayingCardDeck.h"
+#import "PlayingCard.h"
+
+@implementation PlayingCardDeck
+
+- (instancetype)init
+{
+    self = [super init];
+
+    if (self)
+    {
+        for (NSString *suit in [PlayingCard validSuits])
+        {
+            for (NSUInteger rank = 1; rank <= [PlayingCard maxRank]; rank += 1)
```

Comment:
Please check the 10th statement of Obj-C Style guide 😄

Reply:
it says

> 10. Use i += 1 instead of i++

and that's what I'm doing here